packages/suite/src/actions/settings/constants/walletSettings.ts

```diff
@@ -3,3 +3,4 @@ export const SET_HIDE_BALANCE = '@wallet-settings/hide-balance';
 export const CHANGE_NETWORKS = '@wallet-settings/change-networks';
 export const SET_LAST_USED_FEE_LEVEL = '@wallet-settings/set-last-used-fee-level';
 export const FROM_STORAGE = '@wallet-settings/from-storage';
+export const SET_BITCOIN_UNITS = '@suite/set-btc-units';
```

Comment:
```suggestion
export const SET_AMOUNT_UNIT = '@suite/set-amount-unit';
```
let's keep consistent naming